corporate/views/support.py

```diff
@@ -0,0 +1,68 @@
+from urllib.parse import urlencode, urljoin, urlunsplit
+
+from django import forms
+from django.conf import settings
+from django.http import HttpRequest, HttpResponse
+from django.shortcuts import render
+from django.urls import reverse
+
+from zerver.decorator import zulip_login_required
+from zerver.lib.request import has_request_variables
+from zerver.lib.send_email import FromAddress, send_email
+from zerver.models import get_realm
+
+
+class SupportRequestForm(forms.Form):
+    MAX_SUBJECT_LENGTH = 100
+    request_subject = forms.CharField(max_length=MAX_SUBJECT_LENGTH)
+    request_message = forms.CharField(widget=forms.Textarea)
```

Comment:
What's the thinking around the choice of maximum length here?  I'd copy what GitHub uses for their "contact support" form and note it in a comment.